Cargo.toml

```diff
@@ -3,7 +3,8 @@ name = "dusk-schnorr"
 version = "0.16.0"
 authors = [
   "Luke Pearson <luke@dusk.network>", "zer0 <matteo@dusk.network>",
-  "Victor Lopez <victor@dusk.network>", "CPerezz <carlos@dusk.network>"
+  "Victor Lopez <victor@dusk.network>", "CPerezz <carlos@dusk.network>",
```

Comment:
I think that for some time now Dusk opted for eliminating authors from the libraries since the contributions of each contributor can be traced. Just yesterday we discussed it with Matteo and he said that we can take advantage of the PRs to eliminate all the authors, so that the old repos become consistent with the new policy implemented in the new ones! So, I suggest eliminating all authors @ZER0